### ISSUE TITLE
Pain with passive flexion at joint

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10786]
**Preferred term label:**
Pain with passive flexion at joint

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pain in extremity worsens with joint is passively flexed

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Compartment syndrome, acute

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001148
- label: Pain exacerbated by passive flexion at joint
- parents: [Pain characteristic (HP:0025280)](https://hpo.jax.org/app/browse/term/HP:0025280)
- pmid: PMID:25878982
- def: Pain in extremity that worsens with joint is passively flexed.
- comment: This feature may be observed in acute compartment syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10786
- synonyms:
